Automattic/Gravatar-SDK-Android
# QuickEditor: No avatar seleted alert banner

Closes #443 

### Description

This PR adds an informative banner to encourage users to select an avatar instead of using the default one. Per the definition, the banner should be dismissable but must reappear if the condition occurs again. (Ex: selecting an avatar and deleting it).

https://github.com/user-attachments/assets/0b4f3727-3d46-4d86-a00e-6821e9c880d9

### Testing Steps

**Note:** As the deletion flow is not working yet in the SDK, you'll need a browser to remove the selected avatar.

1. Using an account without a selected avatar, open the QE.
2. Verify you can see the alert banner
3. Tap in the X to dismiss the banner
4. Verify the banner is dismissed
5. Close the QE and open it again
6. Verify the banner is visible again
7. Upload a new image 
8. Verify the banner disappears.
9. Close the QE
10. Using the browser, remove the last uploaded (and selected) avatar
11. Open the QE and select any of the already uploaded avatars
12. Verify the banner disappears.

Please verify the UI in both light and dark themes.
